addon.xml

```diff
@@ -27,14 +27,16 @@
   <extension point="xbmc.addon.metadata">
     <platform>all</platform>
     <language>en</language>
-    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 3, 29 June 2007</license>
```

Comment:
This can't be done unless its explicitly "or later".

Reply:
Just for keeping the information in one place; I simply updated this value to the current license from the repo. The license file change was done upstream back in january.